app/vlinsert/opentelemetry/opentelemetry.go

```diff
@@ -136,7 +136,24 @@ func pushFieldsFromScopeLogs(sc *pb.ScopeLogs, commonFields []logstorage.Field,
 			Name:  "severity",
 			Value: lr.FormatSeverity(),
 		})
-
+		if lr.Flags > 0 {
+			fields = append(fields, logstorage.Field{
+				Name:  "_flags",
```

Comment:
hey @bzeron 
what is reason to add underscores to names?

Reply:
just like _msg, you can suggest how to name this?

Reply:
just remove underscore prefixes

Reply:
@AndrewChubatiuk The naming has been modified and completed